data/hails.txt

```diff
@@ -2721,6 +2748,7 @@ phrase "hostile Syndicate"
 		"Thank you for volunteering to be a target ship for Syndicated Weapon Systems!"
 		"Syndicated Systems disapproves of space littering; your ship's wreckage will be recycled after we are finished destroying it."
 		"With authority vested in me by Syndicated Systems corporate charter, section 8, paragraph 22... prepare to die!"
+                "The remains of your soon to be derelict ship will pad my profit margins!"
```

Comment:
Enter the github online "edit" mode for this file. Select the whitespace behind any of the other Syndicate hails, and you'll see a different color for one character, which I think is a different whitespace character. Then do the same selection behind my new one. You'll see it doesn't have the same additional (different?) character. This is causing them to not line up in some of github's views.